cwms-data-api/build.gradle

```
@@ -246,7 +246,7 @@ task run(type: JavaExec) {
 }
 
 task integrationTests(type: Test) {
-    dependsOn test
+//    dependsOn test
```

`dependsOn` test is part of the fail fast scheme we use, if the non-integration tests are failing something is wrong that will very likely affect the integration tests.

---

That was commented in order to speed up test runtime on my machine and wasn't meant to be pushed to git. I've since fixed that.

---

@MikeNeilson